README.md

```diff
@@ -15,10 +15,10 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
-| mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
-| owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
-| install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
+| terraform\_group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
+| terraform\_mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
+| terraform\_owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
+| terraform\_install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
```

Comment:
Put variables in alphabetical order:
```suggestion
| terraform\_group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
| terraform\_install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
| terraform\_mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
| terraform\_owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
```

Reply:
Addressed in [09c15c2](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/09c15c20497c773232ce68946159ed43ba60b386) - also moved terraform_docs_version